src/App.vue

```diff
@@ -26,6 +46,12 @@ export default defineComponent({
     </template>
   </metainfo>
   <div class="app">
+    <div v-if="isError" class="app__error">
+      <span class="app__error-info">{{ errorMessage }}</span>
+      <button class="app__error-escape-button" @click.prevent="this.mainStore.error = null">
```

Comment:
No need to prevent the default behavior here 😄 
```suggestion
      <button class="app__error-escape-button" @click="this.mainStore.error = null">
```